src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -2886,10 +2888,12 @@ namespace InternalHeatGains {
                             thisZoneITEq.Class = ITEClassB;
                         } else if (UtilityRoutines::SameString(AlphaName(10), "C")) {
                             thisZoneITEq.Class = ITEClassC;
+                        } else if (UtilityRoutines::SameString(AlphaName(10), "H1")) {
+                            thisZoneITEq.Class = ITEClassH1;
                         } else {
```

Comment:
This is the code I am talking about re: `getEnumerationValue`. Also, there is no need to print this error because the epJSON parser already does for these types of errors (you can verify). You can simply set `ErrorsFound ||= (thisZoneITEq.Class == ITEClass::Invalid);`